quickstart/include/sources.hpp

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: Copyright (c) 2020-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <srf/node/rx_source.hpp>
+
+namespace srf::ext::quickstart {
+
+class IntSource : public node::RxSource<int>
+{
+  public:
+    IntSource();
```

Comment:
Not that it matters, but you could provide the number of ints as an argument to the constructor.
```suggestion
    IntSource(int count);
```

Reply:
Yes, the tests do this. I'll leave it an exercise to the reader to adapt the quickstart guide for now.